src/ts/components/subtitleoverlay.ts

```diff
@@ -67,9 +69,14 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
         this.subtitleContainerManager.removeLabel(this.previewSubtitle);
       }
 
-      this.subtitleContainerManager.addLabel(labelToAdd);
-      this.updateComponents();
       this.show();
```

Comment:
Why moving after show

Reply:
In order to get the relative sizes and positioning calculations of the `VTTRegion` we have to show the element in the DOM to get those relative values as pixels for calculations